manifest.yaml

```diff
@@ -46,6 +46,9 @@ arguments:
   opslevel.confluenceSpaceKey:
     type: string
     description: The confluence space key for your team (as an example, for developer tooling it is DTSS).
+  opslevel.datadogDashboards.mainLink:
```

Comment:
https://github.com/getoutreach/stencil-opslevel/blob/main/manifest.yaml#L23

we'll just want to use this path (just remove the `opslevel.` prefix) - other than that it LGTM

Reply:
Hmm, I will need to test that.

Reply:
Couldn't figure out how to test that in practice but the change makes sense.